config/printer-creality-ender3-s1-2021.cfg

```diff
@@ -0,0 +1,130 @@
+#Compile Firmware with:
+#STM32 => STM32F103
+#Bootloader offset 28KiB
+#Clock Reference 8MHz
+#Communication interface USART1 PA10/PA9 for using built-in USB-Port
+
+[mcu]
+serial: /dev/serial/by-id/usb-1a86_USB_Serial-if00-port0
+restart_method: command
+
+[stepper_x]
+step_pin: PC2
+dir_pin: PB9
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: ^!PA5
+position_endstop: 0
+position_max: 240
+homing_speed: 50
+
+[stepper_y]
+step_pin: PB8
+dir_pin: PB7
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: ^!PA6
+position_endstop: 0
+position_max: 233
+homing_speed: 50
+
+[stepper_z]
+step_pin: PB6
+dir_pin: !PB5
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 8
+endstop_pin: probe:z_virtual_endstop
+position_min: -5
+position_max: 272
+
+[safe_z_home]
+home_xy_position:154.10,158.50
+speed: 100
+z_hop: 5
+z_hop_speed: 5
+
+[bltouch]
+sensor_pin: ^PC14
+control_pin: PC13
+x_offset: -36.6
+y_offset: -41
+z_offset: 0
+speed: 5.0
+stow_on_each_sample: false
+
+[bed_mesh]
+speed: 120
+horizontal_move_z: 5
+mesh_min: 15, 15
+mesh_max: 206.6,190
+probe_count: 5,5
+fade_start: 1
+fade_end: 10
+fade_target: 0
+mesh_pps: 2,2
+algorithm: bicubic
+bicubic_tension: 0.2
```

Comment:
Are these settings needed?
`fade_start`, `mesh_pps: 2,2`, `bicubic_tension: 0.2` are default anyway and as such should not be repeated in the config.
In general fading can lead dimensional errors in Z

Reply:
generally i though it should be a real config and thats my config that i using. and if fading anyway enabled by default i could remove it here.

Reply:
> generally i though it should be a real config

The config should contain what is needed to get the printer working safely and with its default options. Fading is rather something that can be used according to personal preferences and models that are printed.
If not strictly needed for this type of printer, then please remove it.